features/FEATURE_PSA/TARGET_TFM/src/os_wrapper_cmsis_rtos_v2.c

```diff
@@ -11,31 +11,59 @@
 
 #include <string.h>
 #include "cmsis_os2.h"
+#include "rtx_os.h"
+#include "stdlib.h"
 
 /* This is an example OS abstraction layer for CMSIS-RTOSv2 */
 
 void *os_wrapper_thread_new(const char *name, int32_t stack_size,
                             os_wrapper_thread_func func, void *arg,
                             uint32_t priority)
 {
-    osThreadAttr_t task_attribs = {.tz_module = 1};
+    osRtxThread_t *thread;
+    void *stack;
 
-    /* By default, the thread starts as osThreadDetached */
-    if (stack_size != OS_WRAPPER_DEFAULT_STACK_SIZE) {
-        task_attribs.stack_size = stack_size;
+    thread = (osRtxThread_t *)malloc(osRtxThreadCbSize);
+    if(thread == NULL){
```

Comment:
Does this code pass AStyle? Is AStyle being run on this code?

Both Mbed OS and TF-M style is to have a space after keywords like `if`.

Reply:
This file is in the ignore list in `.astyleignore`
`features/FEATURE_PSA/TARGET_TFM`
Since we are going to maintain it, I can format it, but it can't be caught by the CI, because the whole folder is ignored.
So let me know your thoughts.

Reply:
Fixed.

Reply:
We should use Mbed OS style and change the .astyleignore to ensure we have CI checks on style.

Reply:
I'm not sure how this can be achieved
The importer uses this json - `tools/psa/tfm_ns_import.json` to not only import individual files but folders too, therefore **.astyleignore** completely ignores `features/FEATURE_PSA/TARGET_TFM`
So, is there a mechanism in mbed-os to check a particular file for astyle where a complete directory is ignored for check?

Reply:
Modify .astyleignore to not ignore the entire folder. Add back all folders and files you do want ignored.

Reply:
Fixed